_stbt/gst_utils.py

```diff
@@ -107,6 +107,14 @@ def __array_finalize__(self, obj):
             return
         self.time = getattr(obj, 'time', None)  # pylint: disable=attribute-defined-outside-init
 
+    def __repr__(self):
+        if len(self.shape) == 3:
+            dimensions = "%dx%dx%d" % (
+                self.shape[1], self.shape[0], self.shape[2])
+        else:
+            dimensions = "%dx%d" % (self.shape[1], self.shape[0])
+        return "<stbt.Frame(time=%r, dimensions=%s)>" % (self.time, dimensions)
```

Comment:
`"shape=%r" % (self.shape,)`?

Reply:
Numpy shapes are (height, width) which is confusing for people who aren't dealing with numpy directly.